app/views/notifications_mailer/document_list.text.erb

```diff
@@ -1,3 +1,7 @@
 Hi <%= @user.name %>,
 
-Please find attached the CSV of <%= @document_klass.title.pluralize %><% if @query.present? %> matching the query "<%= @query %>"<% end %> you requested from GOV.UK specialist publisher.
+You requested a CSV of <%= @document_klass.title.pluralize %><% if @query.present? %> matching the query "<%= @query %>"<% end %> from GOV.UK specialist publisher.
+
+You can download this from the following address:
+
+<%= @url %>
```

Comment:
It would be good to say something in the commit message about why this change in general " Upload csv to s3 and link in email" is happening.